pandas/plotting/_core.py

```diff
@@ -1,20 +1,20 @@
 import importlib
-from typing import List, Type  # noqa
 import warnings
 
+from pandas._config import get_option
+
 from pandas.util._decorators import Appender
 
 from pandas.core.dtypes.common import is_integer, is_list_like
 from pandas.core.dtypes.generic import ABCDataFrame, ABCSeries
 
-import pandas
 from pandas.core.base import PandasObject
 
 # Trigger matplotlib import, which implicitly registers our
 # converts. Implicit registration is deprecated, and when enforced
 # we can lazily import matplotlib.
 try:
```

Comment:
Can you replace this block with `compat._optional.import_optional_dependency`?

Reply:
good idea